Makefile

```diff
@@ -67,7 +70,8 @@ help:
 	@echo "clean     - remove build directory and all simulation artefacts"
 	@echo ""
 	@echo "Default simulator is Icarus. To use another set environment variable SIM as below"
-	@for X in $(shell ls cocotb/share/makefiles/simulators/); do \
-		echo $$X | sed 's/^[^.]*./export SIM=/';\
+	@echo "Available simulators: $(AVAILABLE_SIMULATORS)"
+	@for X in ${AVAILABLE_SIMULATORS}; do \
```

Comment:
Shouldn't this be
```suggestion
	@for X in $(AVAILABLE_SIMULATORS); do \
```

Reply:
Both work, but for consistency this is indeed nicer.